gandi/cli/commands/domain.py

```diff
@@ -58,8 +57,31 @@ def info(gandi, resource):
 @pass_gandi
 def create(gandi, domain, duration, owner, admin, tech, bill, background):
     """Buy a domain."""
-    result = gandi.domain.create(domain, duration, owner, admin, tech, bill,
-                                 background)
+
+    # If domain is not available, no need to continue
+    if not domain:
+            return
+
+    user_handle = gandi.call('contact.info')['handle']
```

Comment:
you should never call directly a method like `call` which is "internal", use `gandi.contact.info()` instead